docs/resources/gateway_plugin_jwt_signer.md

```diff
@@ -41,21 +41,25 @@ resource "konnect_gateway_plugin_jwt_signer" "my_gatewaypluginjwtsigner" {
     access_token_introspection_scopes_required = [
       "..."
     ]
-    access_token_introspection_timeout       = 4.24
-    access_token_issuer                      = "...my_access_token_issuer..."
-    access_token_jwks_uri                    = "...my_access_token_jwks_uri..."
-    access_token_jwks_uri_client_certificate = "...my_access_token_jwks_uri_client_certificate..."
-    access_token_jwks_uri_client_password    = "...my_access_token_jwks_uri_client_password..."
-    access_token_jwks_uri_client_username    = "...my_access_token_jwks_uri_client_username..."
-    access_token_jwks_uri_rotate_period      = 0.18
-    access_token_keyset                      = "...my_access_token_keyset..."
-    access_token_keyset_client_certificate   = "...my_access_token_keyset_client_certificate..."
-    access_token_keyset_client_password      = "...my_access_token_keyset_client_password..."
-    access_token_keyset_client_username      = "...my_access_token_keyset_client_username..."
-    access_token_keyset_rotate_period        = 4.53
-    access_token_leeway                      = 0.51
-    access_token_optional                    = false
-    access_token_request_header              = "...my_access_token_request_header..."
+    access_token_introspection_timeout = 4.24
+    access_token_issuer                = "...my_access_token_issuer..."
+    access_token_jwks_uri              = "...my_access_token_jwks_uri..."
+    access_token_jwks_uri_client_certificate = {
+      id = "...my_id..."
+    }
+    access_token_jwks_uri_client_password = "...my_access_token_jwks_uri_client_password..."
+    access_token_jwks_uri_client_username = "...my_access_token_jwks_uri_client_username..."
+    access_token_jwks_uri_rotate_period   = 0.18
+    access_token_keyset                   = "...my_access_token_keyset..."
+    access_token_keyset_client_certificate = {
+      id = "...my_id..."
```

Comment:
This was broken before?
I see based on plugin schema this is supposed to be of `foreign` type